### DISCUSSION TITLE
Encountered error while creating the IPA: error: exportArchive: Failed to login

### DISCUSSION THREAD

**Discussion by omi-jbrivero** [codemagic-ci-cd/codemagic-docs#1351] [Q&A] [answered]
Hello All, 

Encountered error while creating the IPA: error: exportArchive: Failed to login
try distributing the app in Xcode: "open/xxx/buidlers/clone/xxxx/build/ios/archive/Runner.xcarchive"

Hope for your kind help for this one. 

Thank you

**Comment by himesh-cm** [comment]
Hi @jasonbrianrivero , Can you please share the build id, so we can look into your query?

**Reply by omi-jbrivero** [reply]
63d31d021253b33c478e275e

**Reply by himesh-cm** [reply]
Thread continued on intercom, will share the results here once its resolved.

**Comment by himesh-cm** [comment, marked as answer]
You have
export-options-plist=./ExportOptionsPlist.prod.plist \

The default path where it gets saved is /Users/builder/export_options.plist 
can you please change this to 
```
--export-options-plist=/Users/builder/export_options.plist
```
and try running the build again.
